INSTALL.md

```diff
@@ -15,18 +15,32 @@ and under [Arch/Manjaro Linux](#preparing-for-the-installation-under-arch-linux-
 
 ### Preparing for the installation under Mac OS X
 
-NB: The method explained below is recommended for beginners.
-A more flexible, but complex, installation method is given in [INSTALL\_OPAM.md](./INSTALL_OPAM.md).
+NB: The following is a recommended simple approach.
+For more customisation of your installation, see [INSTALL\_OPAM.md](./INSTALL_OPAM.md).
 
-1. Install "Homebrew", available from http://brew.sh/.
-2. Using Homebrew, install ocaml with the following command:
+1. Install OCaml and required libraries, using your preferred package manager.  E.g. using Homebrew (http://brew.sh/), issue the command:
 ```bash
-$ brew install objective-caml ocaml-num camlp5 bash ocaml-findlib
+$ brew install ocaml opam ocaml-num ocaml-findlib
 ```
-3. Install Emacs from https://emacsformacosx.com/.
-  
-Now proceed with [Installation of ProofGeneral](#installation-of-proofgeneral-all-operating-systems) and [Installing UniMath](#installing-unimath) below.
+2. If the build script later asks you to switch to a specific OCaml version, and/or install extra packages, you can do this with:
+```bash
+$ opam switch 4.11.0
```

Comment:
This won't work until after `opam init` is run.  And after that I get this error:
```sh
$ opam switch 4.11.0
[ERROR] No switch 4.11.0 is currently installed. Did you mean 'opam switch create 4.11.0'?
        Installed switches are:
          - default

```

Reply:
After running that command, it proceeds up to this error:
```sh
...
OCAMLOPT -a -o vernac/vernac.cmxa
OCAMLOPT -a -o stm/stm.cmxa
OCAMLOPT -a -o toplevel/toplevel.cmxa
OCAMLC    kernel/byterun/coq_fix_code.c
OCAMLC    kernel/byterun/coq_memory.c
OCAMLC    kernel/byterun/coq_values.c
OCAMLC    kernel/byterun/coq_interp.c
cd kernel/byterun/ && \
"/usr/local/bin/ocamlfind" ocamlmklib -oc coqrun coq_fix_code.o coq_memory.o coq_values.o coq_interp.o
COQMKTOP -o bin/coqtop.opt
File "topbin/coqtop_bin.ml", line 11, characters 20-32:
11 | let drop_setup () = Mltop.remove ()
                         ^^^^^^^^^^^^
Error: Unbound module Mltop
make[2]: *** [Makefile.build:422: bin/coqtop.opt] Error 2
make[2]: Leaving directory '/Users/dan/src/ProofChecking/UniMath/UniMath.git/sub/coq'
make[1]: *** [Makefile.make:178: submake] Error 2
make[1]: Leaving directory '/Users/dan/src/ProofChecking/UniMath/UniMath.git/sub/coq'
Makefile:76: .coq_makefile_output.conf: No such file or directory
make: *** [Makefile:244: sub/coq/bin/coq_makefile] Error 2
```

Reply:
Re `opam switch create` — thanks, I’d been sloppy transcribing my notes.  Fixed, hopefully.

Regarding your `Error: Unbound module Mltop`: Did you manage to troubleshoot this?  My guess would have been files left around from previous compilations, so I’d try running `git clean -Xfd` and `git submodule foreach git clean -Xfd` and then re-running `make` (but of course you know that as well as I do).  If that fixes it, then I don’t think we need to mention it here — it’s unlikely to occur for first-time users, not OS-specific, and we already mention `git clean` as a troubleshooting tip below.  If it seems more OS-specific though then yes, we should try to pre-emptively avert it or give a fix!

Reply:
No, at that point I just stopped.  It looks vaguely familiar.  I'll try to troubleshoot it.

(I did get cleans, as usual.)